network.tf

```diff
@@ -6,7 +6,7 @@ module "virtual_network_connections" {
   virtual_network_connections = {
     for key, vnet_conn in var.virtual_network_connections : key => {
       name                      = vnet_conn.name
-      virtual_hub_key           = vnet_conn.virtual_hub_key
+      virtual_hub_id            = module.virtual_hubs.resource_id[vnet_conn.virtual_hub_key]
```

Comment:
How was this working before? Was it broken? Looks like there was a refactor to submodules.

Reply:
I see the problem, none of the tests were run for this PR... https://github.com/Azure/terraform-azurerm-avm-ptn-virtualwan/pull/133

Reply:
So v0.7.0 doesn't actually work...